Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -8465,15 +8465,14 @@ void WebPage::requestTextRecognition(Element& element, TextRecognitionOptions&&
         return;
     }
 
-    auto* renderer = element.renderer();
-    if (!is<RenderImage>(renderer)) {
+    auto* renderImage = dynamicDowncast<RenderImage>(element.renderer());
```

Comment:
Would be nice to use a CheckedPtr here.

Reply:
I found a near identical case elsewhere in the file. Will update both.